test_app/tests/lib/utils/test_db.py

```diff
@@ -14,12 +14,16 @@ def test_migrations_are_complete():
     assert migrations_are_complete()
 
 
-class TestAdvisoryLock:
+class SkipIfSqlite:
```

Comment:
Out of curiosity did you have to make a parent class here? Could you have just made a fixture at the "root" of the python file that both classes would have picked up? Also, could this be useful somewhere else (i.e. should we make this available from a normal fixture)?

Reply:
> Could you have just made a fixture at the "root" of the python file that both classes would have picked up?

If making it module scope, but I'm not a big fan of module scope. I've used it sometimes, but it's better to be explicit IMO, even if more verbose. Not everything in this module should skip sqlite, at least philosophically.